src/Root.tsx

```diff
@@ -0,0 +1,22 @@
+import { HashRouter, Routes, Route } from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+import { HomeRedirect } from './pages/HomeRedirect';
+
+export const Root = () => (
+  <HashRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route path="home" element={<HomeRedirect />} />
```

Comment:
Use `Navigate` from `react-router-dom`
```suggestion
        <Route path="home" element={<Navigate to={''..''}/>} />
```